docs/supporting-internet-explorer-8.md

```diff
@@ -0,0 +1,108 @@
+# Supporting Internet Explorer 8
+
+If you are including GOV.UK Frontend as part of your application's stylesheets
+then you will need to do some additional work to support Internet Explorer 8.
```

Comment:
Can you put (IE8) in brackets after Internet Explorer 8? That will make it clearer when we refer to it as IE8 hereafter.